index.js

```diff
@@ -1,32 +1,45 @@
-const client = require('./lib/statsd');
+/*
+ * Initialize a hot-shots client.
+ */
 
-const isNotEmpty = (obj) => obj && Object.getOwnPropertyNames(obj).length;
+const os = require('os');
 
-function keyValuesToString(keyValues) {
-  return Object.keys(keyValues)
-    .reduce((r, k) => r.concat(`${k}=${keyValues[k]}`), [])
-    .join(',');
+const statsdClient = require('hot-shots');
+const strigoLogger = require('strigo-node-logger');
+
+const STATSD_HOST = '127.0.0.1';
+const STATSD_PORT = 8125;
+const DEFAULT_ENV = os.hostname();
```

Comment:
can we remove hostname as env's default? maybe make it a must parameter?

Reply:
I don't see why we can't default to the hostname. It makes sense to me that when someone didn't set an env, they'd at least be able to relate to metrics to the hostname they came from. That's specifically true for local envs, and is what we do with the logger as well.

Reply:
where are the metrics going when running locally?

Reply:
We have an `env` label both in the logger and the metrics framework.

And yes, they're currently dropped. I agree it makes it irrelevant for now.